clients/fides-js/src/lib/initialize.ts

```diff
@@ -442,6 +444,8 @@ export const initialize = async ({
         options,
         overrides?.experienceTranslationOverrides,
       );
+      // eslint-disable-next-line no-param-reassign
+      fides.locale = i18n.locale || DEFAULT_LOCALE;
```

Comment:
had to add this to the window.Fides object so that the GPP extension could have access to it, since it has no i18n capabilities on its own (just event handling). It's needed in the string -> consent process to get the correct history ID for the privacy-consent payload.